qdrant-landing/content/documentation/concepts/hybrid-queries.md

```diff
@@ -1025,259 +1025,142 @@ collection `another_collection`.
+- **constant** - A floating point number. e.g. `0.5`.
+- `"$score"` - Reference to the score of the point in the prefetch. This is the same as `"$score[0]"`.
+- `"$score[0]"`, `"$score[1]"`, `"$score[2]"`, ... - When using multiple prefetches, you can reference specific prefetch with the index within the array of prefetches.
+- **payload key** - Any plain string will refer to a payload key. This uses the jsonpath format used in every other place, e.g. `key` or `key.subkey`. It will try to extract a number from the given key.
+- **condition** - A filtering condition. If the condition is met, it becomes `1.0`, otherwise `0.0`.
+- **mult** - Multiply an array of expressions.
+- **sum** - Sum an array of expressions.
+- **div** - Divide an expression by another expression.
+- **abs** - Absolute value of an expression.
+- **pow** - Raise an expression to the power of another expression.
+- **sqrt** - Square root of an expression.
+- **log10** - Base 10 logarithm of an expression.
+- **ln** - Natural logarithm of an expression.
+- **exp** - Exponential function of an expression (`e^x`).
+- **geo distance** - Haversine distance between two geographic points. Values need to be `{ "lat": 0.0, "lon": 0.0 }` objects.
+- **decay** - Apply a decay function to an expression, which clamps the output between 0 and 1. Available decay functions are **linear**, **exponential**, and **gaussian**. [See more](#boost-points-closer-to-user).
+- **datetime** - Parse a datetime string (see formats [here](/documentation/concepts/payload/#datetime)), and use it as a POSIX timestamp, in seconds.
+
+It is possible to define a default for when the variable (either from payload or prefetch score) is not found. This is given in the form of a mapping from variable to value. 
+If there is no variable, and no defined default, a default value of `0.0` is used.
 
-```java
-import static io.qdrant.client.ConditionFactory.matchKeyword;
-import static io.qdrant.client.QueryFactory.nearest;
-import static io.qdrant.client.QueryFactory.orderBy;
+<aside role="status">
+    
+**Considerations when using formula queries:**
+
+- Formula queries can only be used as a rescoring step.
+- Formula results are always sorted in descending order (bigger is better). **For euclidean scores, make sure to negate them** to sort closest to farthest.
+- If a score or variable is not available, and there is no default value, it will return an error.
+- If a value is not a number (or the expected type), it will return an error.
+- To leverage payload indices, single-value arrays are considered the same as the inner value. For example: `[0.2]` is the same as `0.2`, but `[0.2, 0.7]` will be interpreted as `[0.2, 0.7]`
+- Multiplication and division are lazily evaluated, meaning that if a 0 is encountered, the rest of operations don't execute (e.g. `0.0 * condition` won't check the condition).
```

Comment:
Does it make sense to add a note about performance?

- use indexed payload values
- less payload usage in formulas means more performant formulas
- more performant formulas means faster search